### DISCUSSION TITLE
iframe doesn't work

### DISCUSSION THREAD

**Discussion by Seondo** [Dhruv-Techapps/acf-docs#443] [Q&A]
Tried Iframe writing, but doesn't work. can you guys check my console ?

![1](https://github.com/user-attachments/assets/ac16e49f-5920-42f0-8baa-0ad7ba9d7417)
![20241111_123643](https://github.com/user-attachments/assets/4e2a3767-32f3-4b00-9742-93a775d407d5)
![20241111_123650](https://github.com/user-attachments/assets/023842a0-cb4d-4e33-9c91-d220989fb60a)
![20241111_123700](https://github.com/user-attachments/assets/ff044863-dbcb-4da9-9d6b-6d793c028249)

**Comment by dharmesh-hemaram** [comment]
We have developed and deployed fix on DEV version it will be available on BETA, STABLE soon with version number v4.0.42

https://github.com/Dhruv-Techapps/acf-docs/issues/444
